packages/dbml-core/src/parse/ANTLR/ASTGeneration/PostgresASTGen.js

```diff
@@ -0,0 +1,941 @@
+/* eslint-disable class-methods-use-this */
+import _ from 'lodash';
```

Comment:
Please import only the `lodash` functions that you are using. For example:
```javascript
import { map } from 'lodash';
```

Reply:
Fixed in https://github.com/holistics/dbml/pull/416/commits/15d016f623f2d2c5bcf6a5ab90510d1f4723b782.